sdk/typescript/src/verify/index.ts

```diff
@@ -53,26 +57,38 @@ export async function verifyTransactionBlock(
 	return parsedSignature.publicKey;
 }
 
-function parseSignature(signature: SerializedSignature) {
+export function parseSignature(signature: SerializedSignature) {
```

Comment:
This method is pretty useful (at least when writing tests) but we export an existing parseSerializedSignature method from the signature package, which doesn't instantiate the PublicKey class, and exporting both of these without clearer names might be confusing